<launch>
	<node 
		pkg = "sensor"
		type = "start_node.py"
		name = "thc_sensor"
		output = "screen"
	>
		<param name="port" value="/dev/ttyUSB1" />
		<param name="slave_adress" value="16" />
		<param name="baudrate" value="9600" />
		<param name="timeout" value="0.05" />
		<param name="publication_period" value="1" />
	</node>
</launch>
